tests/resources/test_table.py

```diff
@@ -98,6 +98,21 @@ def test_list_table_versions(collection, session):
     assert results[0].uid is not None
 
 
+def test_list_for_ara_definition(collection, session):
```

Comment:
This would also need to change to `test_list_by_ara_definition`

Reply:
Also fixed.